src/components/drawing/index.js

```diff
@@ -579,16 +579,17 @@ drawing.makeTester = function(gd) {
         tester.node()._cache = {};
     }
 
-    gd._tester = tester;
-    gd._testref = testref;
+    drawing.tester = gd._tester = tester;
+    drawing.testref = gd._testref = testref;
```

Comment:
> child windows need their own tester nodes

Do they? It looks to me (fiddling in the [codepen](http://codepen.io/anon/pen/RRpMzL) from @nielsenb-jf ) as though if you have a reference to a node, you can interact with it fully even if it's in a different window. It won't get the same CSS applied to it, sure, but we don't depend on that here, do we?